python/cudf/cudf/core/buffer/spillable_buffer.py

```diff
@@ -461,6 +438,22 @@ def serialize(self) -> Tuple[dict, list]:
                 ]
             return header, frames
 
+    def copy(self, deep: bool = True) -> Self:
+        from cudf.core.buffer.utils import acquire_spill_lock
```

Comment:
*question*: Is there a circular import problem that stops us being able to import `acquire_spill_lock` at the top level?

Reply:
Yes, `utils` import `spillable_buffer`